src/utils/gfycatSdk.js

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import Gfycat from './gfycat-sdk.umd.bundle.min';
+
+export default new Gfycat({
```

Comment:
Since Mattermost is not a single SAAS offering, I'm wondering if this needs to be customizable by the end user deploying gfycat.

Reply:
If we treat it like the YouTube previews, an admin would be able to set up an ID and secret in the System Console under Integrations > External Services

Reply:
I've brought this up to my team.

Reply:
I've added it to System Console under "GIF (Beta)".